content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -782,6 +782,106 @@ APF adds the following two headers to each HTTP response message.
+hold those seats for a shorter duration. List requests can occupy more than 1 
+seat depending on the number of objects fetched during the request. Restricting 
+the number of objects retrieved in a list request (by using pagination or 
+limiting the request to a particular namespace) will use less total seats over a
+shorter period. Furthermore, replacing list requests with watch requests will 
+require lower total concurrency shares as watch requests only occupy 1 seat 
+during its initial burst of notifications.
+
+Keep in mind that queuing or rejected requests from APF could be induced by 
+either an increase in the number of requests or an increase in latency for 
+existing requests. For example, if requests that normally take 1s to execute
+start taking 60s, it is possible that APF will start rejecting requests because 
+requests are occupying seats for a longer duration than normal due to this 
+increase in latency. If APF starts rejecting requests across multiple priority 
+levels without a significant change in workload, it is possible there is an 
+underlying issue with control plane performance rather than the workload or APF 
+settings.
+
+### Priority and fairness settings {#good-practice-apf-settings}
+You can also modify the default FlowSchema and PriorityLevelConfiguration 
+objects or create new objects of these types to better accommodate your 
+workload.
+
+APF settings can be modified to:
+- Give more seats to high priority requests.
+- Isolate non-essential or expensive requests that would starve a concurrency
+level if it was shared with other flows.
+
+#### Give more seats to high priority requests
+1. If possible, the number of seats available across all priority levels for a 
+particular `kube-apiserver` can be increased by increasing the values for the 
+`max-requests-inflight` and `max-mutating-requests-inflight` flags. Alternatively,
+horizontally scaling the number of `kube-apiserver` instances will increase the
+total concurrency per priority level across the cluster assuming there is 
+sufficient load balancing of requests.
+2. You can create a new FlowSchema which references a PriorityLevelConfiguration 
+with a larger concurrency level. This new PriorityLevelConfiguration could be an 
+existing level or a new level with its own set of nominal concurrency shares. 
+Creating a new PriorityLevelConfiguration will reduce the number of seats 
+designated for existing levels. Recall that editing a default FlowSchema or
+PriorityLevelConfiguration will require setting the 
+`apf.kubernetes.io/autoupdate-spec` annotation to false.
+3. You can also increase the NominalConcurrencyShares for the 
+PriorityLevelConfiguration which is serving your high priority requests. 
+Alternatively, for versions 1.26 and later, you can increase the LendablePercent 
+for competing priority levels so that the given priority level has a higher pool 
+of seats it can borrow.
+
+#### Isolate non-essential requests from starving other flows
```

Comment:
I think there's another option to this that is similar: adding a FlowSchema to map traffic to a different default priority level. Most commonly mapping globla-default traffic to something with more shares like workload-low

Reply:
I think this is covered under the second option listed for "Give more seats to high priority requests", which specifies, "2. You can create a new FlowSchema which references a PriorityLevelConfiguration 
with a larger concurrency level. This new PriorityLevelConfiguration could be an 
existing level or a new level with its own set of nominal concurrency shares." To make this clearer, I will provide the global-default to workload-low example.